code/modules/clothing/outfits/ert.dm

```diff
@@ -619,13 +619,7 @@
 	suit_store = /obj/item/gun/ballistic/shotgun/lethal
 	mask = /obj/item/clothing/mask/cigarette/robustgold
 	head = /obj/item/clothing/head/helmet/marine/engineer
-	back = /obj/item/storage/backpack/ert/engineer
-	backpack_contents = list(
```

Comment:
this looks like pretty important gear, you sure they'd rather have a bulky stationary gun and not a fully decked RCD, ammo, and c4s?

Reply:
the rcd i could probably put in their belt, the c4 they never ended up using